src/hooks/useCart/useCart.tsx

```diff
@@ -238,14 +238,14 @@ export const useCart = (
           return { category, quantity, identifiers: identifiers };
         });
 
-      if (numUnverifiedTransactions > 0 || !isUniqueList(identiferValues)) {
-        setError(
-          new Error(
-            `Please enter ${
-              numIdentifiers === 1 ? "code" : "unique codes"
```

Comment:
@szenius i removed the use of `numIdentifiers === 1` condition cos when there were more than one category of vouchers, numIdentifiers can be > 1 but still have blank inputs

Reply:
Eh, I don't really understand what you mean haha!! Can you explain?

Reply:
HAHA ps! cos previous the condition is numIdentifiers === 1 will trigger error for missing input, but actually if we got multiple categories for vouchers (i.e. voucher-1, voucher-2) then if numIdentifiers ===3, it should also trigger error for missing input!

Reply:
changed back to numIdentifiers ===1!